### PR TITLE
Migrate reactions from FireStore to WebSockets cluster

### PR DESCRIPTION
**Pull request by washingtonsoares**

### CODE REVIEW

File: packages/core/src/services/local-storage-api.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { supportsLocalStorage } from 'src/utils/supports';
```
Comment (cesarwbr) [inline packages/core/src/services/local-storage-api.ts:1]: use relative path here

File: packages/core/test/src/auth/user.test.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { User } from "src"
+
+describe('User', () => {
+  test('should return the user instance', () => {
+    const instance = User.instance;
+    expect(instance).toBeInstanceOf(User);
+
```
Comment (cesarwbr) [inline packages/core/test/src/auth/user.test.ts:7]: remove this line

**Review by cesarwbr — CHANGES_REQUESTED** [review]
check the comments below
the build is falling as well

File: packages/chat-sdk/src/channel/channel.ts [hunk 3]
```diff
@@ -729,4 +757,21 @@ export class Channel implements BaseChannel {
...
+    return this.presenceAPI.watchOnlineCount(callback);
+  }
+
+  public watchUserJoined(callback: (user: ExternalUser) => void) {
+    return this.presenceAPI.watchUserJoined(callback);
+  }
+
+  public watchUserLeft(callback: (user: ExternalUser) => void) {
+    return this.presenceAPI.watchUserLeft(callback);
+  }
```
[the remark this answers is not in the document]
Comment (washingtonsoares) [reply packages/chat-sdk/src/channel/channel.ts:774]: Definitely!

File: packages/core/package.json [hunk 4]
```diff
@@ -1,67 +1,64 @@
...
-        "pre-commit": "yarn lint-staged",
-        "pretest": "run-s copy-config:dev",
-        "publish:registry": "yarn publish --access public",
-        "size:check": "run-p size:check:bundle",
-        "size:check:bundle": "cat dist/*.js | gzip -9 | wc -c | awk '{$1=$1/1024; print \"bundle: \",$1,\"kB\";}'",
-        "test": "jest",
-        "test:jest:watch": "jest --watch",
-        "test:watch": "run-s copy-config:dev test:jest:watch"
-    }
+  "name": "@arena-im/core",
```
Comment (washingtonsoares) [inline packages/core/package.json:2]: I think this indentation change is weird, but IDK why it has been changed this way

**Review by cesarwbr — APPROVED** [review]
lgtm

File: packages/core/package.json (hunk 4 above)
Comment (LucasEstreladeOliveira) [reply packages/core/package.json:2]: It seems that nothing changed really, only the indentation, so I don't think we should worry about it

**Review by LucasEstreladeOliveira — APPROVED** [review]
Ok!

**Review by cesarwbr — APPROVED** [review]
lgtm

**Pull request merged**